### DISCUSSION TITLE
Swift bindings to TreeSitter

### DISCUSSION THREAD

**Discussion by ordazgustavo** [tree-sitter/tree-sitter#1045] [Bindings]
Hi! 

A couple of months ago I started working on bindings for [Swift](https://github.com/ordazgustavo/swift-tree-sitter) mostly following the Rust implementation, but I'm currently not able to keep working on them, so I figured I could transfer  the repo to someone from the community that needs them and can continue with the development.

For the most part the bindings are working and even have good test coverage but there are still some rough edges that need to be addressed.

Thanks!

**Comment by QthCN** [comment]
Although I can't work on the swift binding because of my time issue, the swift binding is very stable and very helpful when I used it in a iOS app which can parse and generate symbol infos. Great job 👍

**Comment by maxbrunsfeld** [comment]
Awesome work!
